indra/newview/skins/default/xui/en/menu_fs_poser_poses_btn.xml

```diff
@@ -3,48 +3,32 @@
  layout="topleft"
  name="Settings">
     <menu_item_call
-     label="Load Rotations only"
+     label="Load Selective"
+	 tool_tip="Loads only the joints that are off, and turns them on"
      layout="topleft"
-     name="rotations">
-        <on_click function="Pose.Menu" parameter="rotation"/>
-    </menu_item_call>
-    <menu_item_call
-     label="Load Positions only"
-     layout="topleft"
-     name="positions">
-        <on_click function="Pose.Menu" parameter="position"/>
+     name="rotations_positions">
+        <on_click function="Pose.Menu" parameter="selective"/>
     </menu_item_call>
     <menu_item_call
-     label="Load Scales only"
+     label="Load Selective - Rot Only"
+	 tool_tip="Loads only rotations, only to the joints that are off, and turns them on"
      layout="topleft"
-     name="scale">
-        <on_click function="Pose.Menu" parameter="scale"/>
+     name="rotations_positions">
+        <on_click function="Pose.Menu" parameter="selective_rot"/>
     </menu_item_call>
     <menu_item_separator
      layout="topleft" />
     <menu_item_call
-     label="Load Rotations &amp; Positions"
-     layout="topleft"
-     name="rotations_positions">
-        <on_click function="Pose.Menu" parameter="rot_pos"/>
-    </menu_item_call>
-    <menu_item_call
-     label="Load Rotations &amp; Scales"
-     layout="topleft"
-     name="rotations_scales">
-        <on_click function="Pose.Menu" parameter="rot_scale"/>
-    </menu_item_call>
-    <menu_item_call
-     label="Load Position &amp; Scales"
+     label="Load - Rotations Only"
```

Comment:
What is the difference to `Load Selective - Rot Only`?

Reply:
"Load -Rotations only" loads rotations to all joints, regardless of their on/off state. 
"Load Selective" is a new feature, loading only the joints which are currently 'off'.

I couldn't think of a better name... yeah.

Reply:
"Load Selected (Rot)" (or Rot only) would be a more informative version. Selective just makes you got ok, selective based on what criteria, where as selected is a commonly used term for those bones that are uhm selected.

Reply:
There is lots of space... I can imagine there is a colossal German work for it. 
![image](https://github.com/user-attachments/assets/33ea63ca-ccc3-4544-811c-fbdc86d8c900)

Reply:
only joints that are off? That sounds like the opposite of what I expected, what does off imply here?